### ISSUE TITLE
Font aliasing fault

### ISSUE THREAD

**Issue by chucklepie** [godotengine/godot#44298]
**Godot version:**
3.2.3

**OS/device including version:**
linux mint

**Issue description:**
I cannot find any settings in dynamic font section, I have pixel snap enabled, I have read various issues that say 'fixed', but I cannot find any means of turning off aliasing. 

Below are pictures of the font in Godot and in normal editor at same size showing the problem. As you can see Godot rendering is one ugly thing ;-)

![image](https://user-images.githubusercontent.com/12863685/101913017-6044a780-3bba-11eb-8102-5daf08618f44.png)

![image](https://user-images.githubusercontent.com/12863685/101913055-6c306980-3bba-11eb-8e19-42d09f2363dd.png)

Please let me know there's a simple fix, an aliased font is of no use to anyone in a pixel game. Thanks.

**Steps to reproduce:**
1. use my font attached
2. Set a big font size

[fonts.zip](https://github.com/godotengine/godot/files/5679573/fonts.zip)

**Comment by bruvzg** [comment]
This is known issue with the resource loader (see #28537), instead of loading `.ttf` file directly, create new `DynammicFontData` subresource, uncheck `antialiased` and then set `font path`.

Font resource created this way: [df.tres.zip](https://github.com/godotengine/godot/files/5679665/df.tres.zip)

Diff of original and `df.tres`:
```diff
3,5c3
< [sub_resource type="DynamicFontData" id=3]
< antialiased = false
< font_path = "res://upheavtt.ttf"
---
> [ext_resource path="res://assets/fonts/upheavtt.ttf" type="DynamicFontData" id=1]
12c10
< font_data = SubResource( 3 )
---
> font_data = ExtResource( 1 )
```

<img width="843" alt="Screenshot 2020-12-11 at 16 20 40" src="https://user-images.githubusercontent.com/7645683/101914462-0699a880-3bcd-11eb-90b2-97943be787fd.png">

**Comment by Calinou** [comment]
> I cannot find any settings in dynamic font section, I have pixel snap enabled, I have read various issues that say 'fixed', but I cannot find any means of turning off aliasing.

This is already supported.

- Create a DynamicFont.
- Load a TTF/OTF file into it.
- Edit the DynamicFontData resource, uncheck **Antialiased**.
- Click the "tools" icon at the top right corner of the Inspector and choose **Save As…**. Save the DynamicFontData resource as a `.tres` file. This makes the Antialiased property persist if you run the project. Otherwise, its value will be lost when you ruhn the project.

**Comment by chucklepie** [comment]
Thanks for the swift response, I will try it. I'd suggest that if you have to go down that route to make it work and there is no 'antialias' when loading a TTF, it's not really 'supported' as nobody will know it's there and it's pretty impossible to do it unless you know somebody like you :+1: 

Will it be added as an option in a future patch?

Thanks.

**Comment by chucklepie** [comment]
Hello, it doesn't work, I untick and on saving in the dropdown I get this.

![image](https://user-images.githubusercontent.com/12863685/101916919-7f920380-3bbf-11eb-9a90-284ef76afed6.png)

If I go to the top and click save on the tres
![image](https://user-images.githubusercontent.com/12863685/101916994-959fc400-3bbf-11eb-9a9e-80169f1b479b.png)

It doesn't persist. This is so difficult knowing which section of a resource you have to save to get things to save. You think you've saved a resource, but it has 10 different resources embedded inside that resource and each one persists separately...

**Comment by chucklepie** [comment]
Also, I found the per resource option to disable, and as you can see, absolutely nothing happens :(

![image](https://user-images.githubusercontent.com/12863685/101917542-43ab6e00-3bc0-11eb-8c6a-27c2042ea8f3.png)

**Comment by chucklepie** [comment]
I deleted the font, created a new font resource, redid anti-alias and nothing happened to my font on screen. I tried to save the font data and got the 'resource error' again, I saved the full resource (not just the font data) ok, and on reloading it was gone. Why are there so many levels of saving you have to do when items have multiple resources inside them, it's insane ;)

so, it's still not working in my 3.2.3 with this font.

**Comment by Calinou** [comment]
> I saved the full resource (not just the font data) ok, and on reloading it was gone.

You need to save the DynamicFontData as a resource, not the DynamicFont.

**Comment by chucklepie** [comment]
Yes, this one? All I get is 
![image](https://user-images.githubusercontent.com/12863685/101929308-94c25e80-3bce-11eb-8c43-4e9165b5ae55.png)

I was using DynamicFontData, I just mentioned the other due to erorr messages. But I think I got it sorted but I've possibly found a different bug. Even though I selected clear in the dynamicfont and everything was blank, every time I dragged my ttf into the empty font box I got this is the error log:

Another resource is loaded from path 'res://assets/fonts/standard_font.tres' (possible cyclic resource inclusion).

Nothing else is using it, all scenes are closed and as mentioned I had cleared the data out.

What fixed it was selecting 'clear', then selecting 'make unique', then dropping the font in and clicking save to give it a new name. Apart from it should be happening, I don't know why Godot does not have a 'save as' option instead of the fairly unintuitive way of having to use 'make unique' then 'save'.

thanks for the help.
